packages/core/core/src/BundleGraph.js

```diff
@@ -278,17 +278,11 @@ export default class BundleGraph {
   }
 
   hasParentBundleOfType(bundle: Bundle, type: string): boolean {
-    return (
-      this._graph
-        .getNodesConnectedTo(
-          nullthrows(this._graph.getNode(bundle.id)),
-          'bundle'
-        )
-        .map(node => this._graph.getNodesConnectedTo(node, 'bundle'))
-        .reduce((acc, v) => acc.concat(v), [])
-        .filter(node => node.type === 'bundle' && node.value.type === type)
-        .length > 0
-    );
+    return this._graph
+      .getNodesConnectedTo(nullthrows(this._graph.getNode(bundle.id)), 'bundle')
+      .map(node => this._graph.getNodesConnectedTo(node, 'bundle'))
+      .reduce((acc, v) => acc.concat(v), [])
```

Comment:
`flatMap` 🙂

Reply:
Supported in Node 11+ 😞

Reply:
sorry, should have clarified: `import {flatMap} from '@parcel/utils';`. We have our own 😄